reference_guide/ui_themes/themes_customize.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Customizing UI Themes - Icons and UI Controls
+---
+A UI Theme is customized by adding information to the UI Theme description file that overrides the base (_Light_ or _Darcula_) UI Theme. 
+
+## Introduction to UI Theme Description File Syntax
+The syntax of a Theme description file follows the JSON open-standard file format of key-value pairs. 
+The minimum valid default file is the output of the [DevKit Theme Wizard](themes.md#creating-a-ui-theme-with-the-devkit-theme-wizard). 
+Adding key-value pairs customizes a Theme.
+
+UI Theme key-value pairs often use a color as the `value`.
+Colors are defined by six-digit RGB or eight-digit RGBA hexadecimal notation.
+
+### UI Theme Reference Implementations
+When learning new syntax, it is often useful to have some existing implementations for reference.
+For example, refer to the [Theme description file](upsource:///platform/platform-resources/src/themes/HighContrast.theme.json) for the IntelliJ IDEA _High Contrast_ Theme.
```

Comment:
Not sure how this work but when I clicked this link in the prebuild website sent by Yann, an outdated version of the file was opened https://www.dropbox.com/s/zu0i23tpkt6ch52/Screenshot%202019-01-23%2014.33.38.png?dl=0

Reply:
we're always linking to latest release, so that is 183.4284 (2018.3)